.github/workflows/openapi-docs-push.yml

```
@@ -0,0 +1,19 @@
+name: Test OpenAPI doc is valid (Push)
+
+on:
+  push:
+    paths:
+      - "schemas/v1alpha/openapi.yml"
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: schemas
+    name: Run Spectral
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: stoplightio/spectral-action@v0.8.0
+        with:
+          file_glob: 'schemas/v1alpha/openapi.yml'
+          spectral_ruleset: 'schemas/.spectral.yml'
```

Sorry may have been my bad, now we're in the new `working-directory` we don't need to specify it again:
```suggestion
          file_glob: 'v1alpha/openapi.yml'
          spectral_ruleset: '.spectral.yml'
```